wagn/lib/wagn/generators/wagn/templates/config/routes.erb

```diff
@@ -1,8 +1,6 @@
 # -*- encoding : utf-8 -*-
 
-require 'decko/engine'
-
-Rails.application.routes.draw do
+Wagn.application.routes.draw do
```

Comment:
Sorry, I thought this was in the engine deck template.  I'll make sure I got the locations right.

For the engine deck case, should this now be Decko::Rails.application?  That would still need to reference Rails, but I don't see that would be a problem in that file, right?